main.go

```diff
@@ -53,25 +57,39 @@ func main() {
 	}
 
 	// creating a new AWSclient with the information extracted from the secret file
-	client, err := clientpkg.NewClient(string(accessKeyID), string(secretAccessKey), "", "us-east-1")
+	awsClient, err := clientpkg.NewClient(string(accessKeyID), string(secretAccessKey), "", "us-east-1")
 	if err != nil {
 		fmt.Println("ERROR:", err)
 	}
-	for {
+
+	//integrating the account CRD from the
+	v1alpha1.AddToScheme(client_go_scheme.Scheme)
+
+	// creating a client for reading the AccountID
+	cli, err := client.New(config, client.Options{})
```

Comment:
Does this have to be a different client for the secret reading, or can the same one be used? Let's try and consolidate if we can.

Reply:
clientset is of type " * kubernetes.Clientset"
cli is "client.Client"

they are different data types.

Reply:
removed 'clientset',